src/features/hotspots/setup/HotspotSetupAddTxnScreen.tsx

```diff
@@ -0,0 +1,141 @@
+import { useNavigation } from '@react-navigation/native'
+import React, { useEffect, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useSelector } from 'react-redux'
+import BackScreen from '../../../components/BackScreen'
+import Box from '../../../components/Box'
+import Button from '../../../components/Button'
+import Text from '../../../components/Text'
+import { useConnectedHotspotContext } from '../../../providers/ConnectedHotspotProvider'
+import { RootState } from '../../../store/rootReducer'
+import usePermissionManager from '../../../utils/usePermissionManager'
+import { HotspotSetupNavigationProp } from './hotspotSetupTypes'
+
+const FEE = 40
```

Comment:
What this fee should be is the biggest thing I'm still not clear on 🤦 
It's currently set to `0` if `freeAddHotspot` otherwise `40`.

Reply:
yeah I think I'll need to take a pass through all of that at some point, so as long as the general flow is working for now, let's not worry too much about the exact values. the blockchain won't accept anything that's invalidly constructed anyway

Reply:
Sounds good. 👍